luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -41,12 +41,12 @@ local powerThresholds = {
     {techLevel = 4.5, threshold = 725000}
 }
 
-
+local pveTeamID = scavengerTeam or raptorTeam
 local scavengerTeam = Spring.Utilities.GetScavTeamID()
 local raptorTeam = Spring.Utilities.GetChickenTeamID()
 for _, teamID in ipairs(teamList) do
     local allyID = select(6, Spring.GetTeamInfo(teamID))
-    if select (4, Spring.GetTeamInfo(teamID, false)) then
+    if not pveTeamID and select (4, Spring.GetTeamInfo(teamID, false)) then
```

Comment:
```suggestion
    if teamID ~= scavengerTeam and teamID ~= raptorTeam and select (4, Spring.GetTeamInfo(teamID, false)) then
```
Currently it's testing that `pveTeamID` is not null, not that it matches the current team ID. It also does not support both Scavengers _and_ Raptors being present.